.github/workflows/release-mainnet-desktop.yml

```diff
@@ -55,7 +41,7 @@ jobs:
           node-version: 14
 
       - name: Run yarn install
-        run: yarn install --frozen-lockfile --network-concurrency 1
+        run: yarn install --frozen-lockfile --network-timeout 1000000
```

Comment:
This particular thing is only for windows because it was failing: https://stackoverflow.com/questions/47812779/how-to-increase-the-timeout-for-yarn-install.  We don't have any git repos now and we shouldn't. Just release an alpha version to npm